airflow/operators/sftp_to_gcs.py

```diff
@@ -0,0 +1,178 @@
+#
```

Comment:
Should be in providers/google/cloud according to latest agreements as GCP is the only "proprietary" side of the transfer.

Reply:
I've moved it to `providers/google/cloud`. All related files as well.